packages/geoprocessing/src/components/SegmentControl.tsx

```diff
@@ -60,23 +46,17 @@ export const SegmentControl = (props: SegmentControlProps) => {
           transitionProperty: "all",
           transitionTimingFunction: "cubic-bezier(0.4, 0, 0.2, 1)",
           transitionDuration: "75ms",
-          insetInlineStart: position,
-          width: `${segmentSizes ? segmentSizes[index] : 0}px`,
+          insetInlineStart: `${segmentWidth * index}px`,
+          width: `${segmentWidth - 4}px`,
```

Comment:
What is the significance of subtracting 4 pixels here?   Is there padding or border to compensate for?

Reply:
Yep, it's to account for the container's border, here:
https://github.com/seasketch/geoprocessing/blob/3122c7a89f91603361b497581e70efd60d8914fe/packages/geoprocessing/src/components/SegmentControl.tsx#L38-L39